tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/Qualification.scala

```diff
@@ -34,7 +34,7 @@ import org.apache.spark.sql.rapids.tool.qualification._
 class Qualification(outputDir: String, numRows: Int, hadoopConf: Configuration,
     timeout: Option[Long], nThreads: Int, order: String,
     pluginTypeChecker: PluginTypeChecker, readScorePercent: Int,
-    reportReadSchema: Boolean, printStdout: Boolean) extends Logging {
+    reportReadSchema: Boolean, printStdout: Boolean, uiEnabled: Boolean = false) extends Logging {
```

Comment:
I would remove the uiEnabled option since its not included here.

Reply:
I agree. uiEnabled option and it's related code can be added in a different PR where we integrate the UI.

Reply:
removed the new argument.